src/tap.rs

```diff
@@ -48,11 +54,11 @@ impl TestGroup {
     pub fn diagnostic(&self, msg: &str) {
         serial_println!("# {}", msg);
     }
-
 }
 
 impl Drop for TestGroup {
     fn drop(&mut self) {
-        self.plan()
+        // HACK: split tests
+        serial_println!("EOT");
```

Comment:
Would it be better to put this in a comment? i.e. `self.diagnostic("EOT")`

Reply:
good point